mobile/lib/constants/api.dart

```diff
@@ -34,6 +34,9 @@ class AirQoUrls {
 
   static String get searchHistory =>
       '${Config.airqoApi}/v2/users/searchHistory';
+
+  static String get syncPlatformAccount =>
+      '${Config.airqoApi}/v2/users/syncAnalyticsAndMobile';
```

Comment:
Thanks @BenjaminSsempala for this.
Just curious, which endpoint is this?
Which `auth-service` endpoints are we using for this synchronisation.....if any?

Reply:
@Baalmart 
Yes, I've just raised a subsequent PR for this endpoint in the API repo.
https://github.com/airqo-platform/AirQo-api/pull/2392

Reply:
Thanks @BenjaminSsempala , just curious....what happened to some previous endpoints I had created? 
Were they somehow "out of touch" with reality? More like "wrong tablets"? :)